pom.xml

```diff
@@ -485,7 +486,7 @@
         <version.org.jboss.metadata>16.1.0.Final</version.org.jboss.metadata>
         <version.org.jboss.mod_cluster>2.1.0.Final</version.org.jboss.mod_cluster>
         <version.org.jboss.narayana>7.2.2.Final</version.org.jboss.narayana>
-        <version.org.jboss.narayana.lra>1.0.1.Final</version.org.jboss.narayana.lra>
+        <version.org.jboss.narayana.lra>1.0.2.Final-SNAPSHOT</version.org.jboss.narayana.lra>
```

Comment:
@marcosgopen We'll need to change this to a released version before merging our WildFly clone. And the released version won't work with WildFly until the WildFly MP LRA testsuite has been updated.

Reply:
Please consider keeping this PR open and never merge it here. I would usually prefer to have it merged in wildfly main and then pull that change into our jboss-as repo.
+1 to change the version to a released one for the WildFly PR